server/routes/handlers/helpers/viewWithRecallAndPerson.ts

```diff
@@ -72,5 +72,14 @@ export const viewWithRecallAndPerson =
         res.locals.assessedByUserName = recall.assessedByUserId
       }
     }
+    if (recall.bookedByUserId) {
+      try {
+        const { firstName, lastName } = await getUserDetails(recall.bookedByUserId, res.locals.user.token)
```

Comment:
Can we parallelise this with the other getUserDetails call if possible? Otherwise we're now delaying every page load with 3 sets of calls in sequence